NetBSD Hello World fails to build on 1.25.0

I'm not a NetBSD user so I can't help too much, but here's what travis says: https://travis-ci.org/mattico/test-netbsd/jobs/361710278

I suppose this could also be a bug in [`cross`](https://github.com/japaric/cross) but this error started with 1.25, so I don't think so...

```
error: linking with `x86_64-unknown-netbsd-gcc` failed: exit code: 1
  |
  = note: "x86_64-unknown-netbsd-gcc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-m64" "-L" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e.1y16o1qfye96o7m0.rcgu.o" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e.3rngp6bm2u2q5z0y.rcgu.o" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e.4xq48u46a1pwiqn7.rcgu.o" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e.zkj6zrh601ni23e.rcgu.o" "-o" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e" "/target/x86_64-unknown-netbsd/debug/deps/test_netbsd-9b57e314db02ac7e.crate.allocator.rcgu.o" "-Wl,--gc-sections" "-pie" "-Wl,-z,relro,-z,now" "-nodefaultlibs" "-L" "/target/x86_64-unknown-netbsd/debug/deps" "-L" "/target/debug/deps" "-L" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib" "-Wl,-Bstatic" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libstd-075ab6f2359a82f1.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libpanic_unwind-428f111496747802.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libunwind-fbadb5623ab8cc25.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/liballoc_system-c7afd75849f41e4c.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/liblibc-42a81d0277a7a698.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/liballoc-20751df81794b150.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libstd_unicode-34dbdc699982f60d.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libcore-8e8a91684c5e06e5.rlib" "/rust/lib/rustlib/x86_64-unknown-netbsd/lib/libcompiler_builtins-4e85b1507e729192.rlib" "-Wl,-Bdynamic" "-l" "pthread" "-l" "gcc_s" "-l" "m" "-l" "util"
  = note: /usr/local/lib/gcc/x86_64-unknown-netbsd/5.3.0/../../../../x86_64-unknown-netbsd/bin/ld: /rust/lib/rustlib/x86_64-unknown-netbsd/lib/libstd-075ab6f2359a82f1.rlib(std-075ab6f2359a82f1.std0.rcgu.o): undefined reference to symbol 'execvp'
          /usr/local/x86_64-unknown-netbsd/lib/libc.so.12: error adding symbols: DSO missing from command line
          collect2: error: ld returned 1 exit status
```

Possibly related to https://github.com/rust-lang/rust/issues/45084

--- a/src/librustc_target/spec/netbsd_base.rs
+++ b/src/librustc_target/spec/netbsd_base.rs
@@ -29,6 +29,7 @@ pub fn opts() -> TargetOptions {
         executables: true,
         target_family: Some("unix".to_string()),
         linker_is_gnu: true,
+        no_default_libraries: false,
         has_rpath: true,
         pre_link_args: args,
         position_independent_executables: true,

--- a/src/librustc_target/spec/x86_64_rumprun_netbsd.rs
+++ b/src/librustc_target/spec/x86_64_rumprun_netbsd.rs
@@ -21,7 +21,6 @@ pub fn target() -> TargetResult {
     base.has_rpath = false;
     base.position_independent_executables = false;
     base.disable_redzone = true;
-    base.no_default_libraries = false;
     base.exe_allocation_crate = None;
     base.stack_probes = true;
 

